ufo2fontir/src/source.rs

```diff
@@ -435,6 +435,37 @@ fn glyph_order(
     Ok(glyph_order)
 }
 
+fn units_per_em(
+    designspace: &DesignSpaceDocument,
+    designspace_dir: &Path,
+) -> Result<u16, WorkError> {
+    let mut upems: HashSet<u16> = HashSet::new();
+    for source in designspace.sources.iter() {
+        let ufo_dir = designspace_dir.join(&source.filename);
+        if !ufo_dir.join("fontinfo.plist").exists() {
+            continue;
+        }
+        let fontinfo = load_plist(&ufo_dir, "fontinfo.plist")?;
```

Comment:
for now this is ok, since we are only reading the unitsPerEm from the fontinfo.plist.. but we will sooner or later read more than just that, so it'd make sense to parse it only once instead of for each individual getter such as in this case for `units_per_em(designspace, designspace_dir)`

Reply:
Good catch. Changed to parse all the font info files then pass a vec of them to the upem fn.